config.yml.mustache

```diff
@@ -0,0 +1,58 @@
+version: 2.1
+
+jobs:
+  build:
+    parameters:
+      coq:
+        type: string
+    docker:
+    - image: coqorg/coq:<<parameters.coq>>
+    resource_class: medium
+    environment:
+      OPAMJOBS: 2
+      OPAMVERBOSE: 1
+      OPAMYES: true
+      TERM: xterm
+    steps:
+    - checkout
+    - run:
+        name: Pull submodules
+        command: git submodule update --init --remote
```

Comment:
Is this yet another subtle difference between this template and the other CI templates?

Reply:
I think so… Maybe adding this `git submodule` part is unneeded for the default coq-community templates? at least for the time being (IMHO the three CI templates should be as close as possible semantically one from the other, and later on other PRs could just as well add features that enhance them, in a row?)

Reply:
Travis CI [clones Git submodules by default](//docs.travis-ci.com/user/customizing-the-build#git-submodules). I've added submodule for GitHub Actions.

Reply:
OK! have you checked that this is exactly that command triggered by Travis for git submodule cloning?

(I'm not very submodule-savvy, but e.g. this blog https://mgalgs.github.io/2014/07/03/git-submodule-update-init-remote-recursive-considered-harmful.html mentions the following commands: `git submodule update --init --recursive; git submodule update --remote`)

Reply:
Travis uses `git submodule update --init --recursive`. Should I just drop `--remote`?

Reply:
Maybe yes − this option is documented here:
https://git-scm.com/docs/git-submodule#Documentation/git-submodule.txt---remote ;
it appears to be related to remote-tracking branches (which would be a bit off-topic in the context of CI builds)

Reply:
and this complies with the doc of GitLab CI as well:
https://docs.gitlab.com/ee/ci/git_submodules.html#using-git-submodules-in-your-ci-jobs
so `git submodule update --init --recursive` seems the way to go

Reply:
(actually the GItLab CI page I mentioned suggests two lines)
```
git submodule sync --recursive
git submodule update --init --recursive
```